config/webpack.config.js

```diff
@@ -255,7 +255,9 @@ module.exports = {
       // It is guaranteed to exist because we tweak it in `env.js`
       process.env.NODE_PATH.split(path.delimiter).filter(Boolean),
     ),
-    alias: {},
+    alias: {
+      concaveman: path.join(paths.appSrc, 'utils', 'webShims', 'concavemock.js'),
```

Comment:
Incase you are wondering, this is needed because this module does an unsave-eval. 

Until we implement the narrative interface editor and therefore need to preview it, this is a workaround.

We also intend to v2.0 the preview function which will resolve this the proper way.